mm2src/mm2_main/Cargo.toml

```diff
@@ -22,6 +22,7 @@ enable-solana = []
 default = []
 
 [dependencies]
+serde_with = "3.0.0"
```

Comment:
I think this was added in a previous attempt to fix the issue without rolling back `rmp`. Please remove it.

Reply:
good catch!